### DISCUSSION TITLE
plan for longer post types?

### DISCUSSION THREAD

**Discussion by flbn** [bluesky-social/atproto#1150] [Bluesky Lexicons]
is there a plan to potentially add a new post type for longerform writing? rn, the 'normal' text accounts for this max:

https://github.com/bluesky-social/atproto/blob/af9f11fbde332dfebb8c79e52f009cfd13d08346/lexicons/app/bsky/feed/post.json#L12

it'd be cool to add an indefinitely long text type for blog-style articles? maybe too complicated but just a thought.

**Comment by jem-computer** [comment]
I think we should do this but as a separate post type - super down to jam on a writing tool!

**Comment by flbn** [comment]
awww yea, did you wanna get a PR going sometime soon?

**Comment by odellus** [comment]
Bump

**Reply by surfdude29** [reply]
FYI [whtwnd.com](https://whtwnd.com/) has a lexicon that lets you make blog posts up to 100,000 characters long, although those posts don't appear automatically on Bluesky, you have to post a link manually:

- https://github.com/bluesky-social/atproto/discussions/2501

**Comment by mterenzio** [comment]
Longform discussions happening here
https://atproto.wiki/en/working-groups/long-form and here
https://discord.com/channels/1097580399187738645/1375128241626677372
